common/src/com/intellij/plugins/haxe/compilation/HaxeCompilerError.java

```diff
@@ -196,7 +196,7 @@ private static boolean isInformationalMessage(String message) {
     }
 
     static Pattern pLibraryNotInstalled = Pattern.compile
-        ("Error: Library ([\\S]+) is not installed(.*)");
+        ("Library (\\S+) (is not installed.*)");
```

Comment:
Dropped the square brackets because they were actually causing matching failures.  Also moved the grouping to pick up the "is not installed" part of the message so we didn't have to duplicate that in the code above.